todo/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import Todo from './components/Todo';
+import TodoForm from './components/TodoForm';
+
```

Comment:
great job on keeping your app.js clean like this by keeping all your logic and code in the components. high-level right here